src/layer/ImageOverlay.js

```diff
@@ -47,7 +47,10 @@ export var ImageOverlay = Layer.extend({
 
 		// @option zIndex: Number = 1
 		// The explicit [zIndex](https://developer.mozilla.org/docs/Web/CSS/CSS_Positioning/Understanding_z_index) of the tile layer.
-		zIndex: 1
+		zIndex: 1,
+		// @option zoomPixelated: String = ''
+		// The explicit [zoomPixelated](https://developer.mozilla.org/zh-CN/docs/Web/CSS/image-rendering) of the tile layer.
+		zoomPixelated:"NN"

```

Comment:
Three things here:

1. As implemented now, this is a boolean switch, it's either on or off; the naming also sounds like it's a boolean. I'd either keep the name and make the value `true`/`false`, or change the name to something like `imageRendering`, since that's the name used in CSS
2. The default should not be nearest neighbor, we should keep default as it is now (`image-rendering: auto`)
3. `NN` is not a very descriptive value; if we're not making this a boolean (see point 1 above), I'd prefer `pixelated` or `nearest-neighbor` to keep to the CSS terminology